examples/python/ImageAlign/img_detections_align.py

```
@@ -0,0 +1,105 @@
+            output,
+            f"{int(detection.confidence * 100)}%",
+            (anchor[0] + 8, anchor[1] + 40),
+            cv2.FONT_HERSHEY_TRIPLEX,
+            0.5,
+            TARGET_COLOR,
+        )
+    return output
+
+device = dai.Device()
+if device.getPlatform() == dai.Platform.RVC2:
+    raise RuntimeError(f"GenericAlign is not supported on the RVC2 platform.")
```

Ruff (0.15.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)